pp-mock/services/data/tripsv3_basic.json

```diff
@@ -84,5 +84,69 @@
 			"seconds": 1740000000,
 			"nanos": 0
 		}
+	},
+	{
+		"supplier_code": {
+			"supplier_code": "DB",
+			"supplier_number": 5483
+		},
+		"segments": [
+			{
+				"segment_id": "SEG8321",
+				"provider_code": "DB",
+				"product_code": {
+					"code": "RE3-89875",
+					"type": 5
+				},
+				"departure": {
+					"date_time": {
+						"seconds": 1742371437,
+						"nanos": 0
+					},
+					"location" : {
+						"location_code" : {
+							"code" : "DUS",
+							"code_type" : "LOCATION_CODE_TYPE_IATA_CODE"
```

Comment:
Are you sure that in the train example we should use IATA codes? 
IATA codes are used for airports - train stations have different codes.
See https://buf.build/chain4travel/camino-messenger-protocol/docs/ffe2c3741a334864983984d0ef6ea39d:cmp.types.v2#cmp.types.v2.LocationCode

Reply:
Now using 3ALPHA codes for train station locations.